aptos-move/framework/aptos-framework/sources/aptos_account.move

```diff
@@ -138,38 +150,54 @@ module aptos_framework::aptos_account {
 
     public fun assert_account_is_registered_for_apt(addr: address) {
         assert_account_exists(addr);
-        assert!(coin::is_account_registered<AptosCoin>(addr), error::not_found(EACCOUNT_NOT_REGISTERED_FOR_APT));
+        assert!(
+            coin::is_account_registered<AptosCoin>(addr),
+            error::not_found(EACCOUNT_NOT_REGISTERED_FOR_APT),
+        );
     }
 
     /// Set whether `account` can receive direct transfers of coins that they have not explicitly registered to receive.
-    public entry fun set_allow_direct_coin_transfers(account: &signer, allow: bool) acquires DirectTransferConfig {
+    public entry fun set_allow_direct_coin_transfers(
+        account: &signer, allow: bool
+    ) acquires DirectTransferConfig {
         let addr = signer::address_of(account);
         if (exists<DirectTransferConfig>(addr)) {
             let direct_transfer_config = borrow_global_mut<DirectTransferConfig>(addr);
             // Short-circuit to avoid emitting an event if direct transfer config is not changing.
-            if (direct_transfer_config.allow_arbitrary_coin_transfers == allow) {
-                return
-            };
+            if (direct_transfer_config.allow_arbitrary_coin_transfers == allow) { return };
```

Comment:
Personally, I'm not for single lining for things like this, but it works.

Would probably keep the old one

Reply:
In that case, should all branch statements with braces, all break to a new line after the '{'?